Tools/NTupleReader.h

```diff
@@ -664,12 +664,12 @@ class NTupleReader
         else if(convertHackActive_ && intuple) //else check if it is a vector<float> or vector<double>
         {
             //hack to get vector<double> as vector<float>, requires DuplicateFDVector() to be run
-            char typen;
+            std::string typen;
             if( typeid(typename std::remove_pointer<T>::type) == typeid(std::vector<float>) && tuple_iter->second.type == typeid(std::vector<double>))
                 typen='f';
             if( typeid(typename std::remove_pointer<T>::type) == typeid(std::vector<double>) && tuple_iter->second.type == typeid(std::vector<float>))
                 typen='d';
-            std::string newname = var+"___" + typen;
+            std::string newname = var + "___" + typen;
```

Comment:
instead of the above, do `std::to_string(typen)`

Reply:
This was a spacing change. It works if typen is a char or a string.